test/controllers/organization_memberships_controller_test.exs

```diff
@@ -55,23 +54,21 @@ defmodule CodeCorps.OrganizationMembershipControllerTest do
     end
 
     test "filters resources by membership id", %{conn: conn} do
-      membership_1 = insert(:organization_membership)
-      membership_2 = insert(:organization_membership)
+      [membership_1, membership_2] = insert_pair(:organization_membership)
       insert(:organization_membership)
 
       params = %{"filter" => %{"id" => "#{membership_1.id},#{membership_2.id}"}}
-      conn = get conn, organization_membership_path(conn, :index, params)
-      data = json_response(conn, 200)["data"]
-      assert data |> length == 2
+      response = conn
+        |> get(organization_membership_path(conn, :index, params))
+        |> json_response(200)
 
-      [first_result, second_result] = data
-      assert first_result["id"] == "#{membership_1.id}"
-      assert second_result["id"] == "#{membership_2.id}"
+      assert response["data"] |> length == 2
```

Comment:
@JoshSmith Thoughts on whether this assertion is redundant? If we assert that we get 2 ids back below, isn't that same as asserting the number of records?